components/datetime/src/provider/gregory/mod.rs

```diff
@@ -34,17 +36,13 @@ pub struct DatePatternsV1 {
     /// By default a locale will prefer one hour cycle type over another.
     pub preferred_hour_cycle: pattern::CoarseHourCycle,
 
-    pub datetime: patterns::DateTimeFormatsV1,
+    pub date_time: patterns::LengthPatternsV1,
```

Comment:
Suggestion: The name would be a lot less ambiguous if it included the term "length", rather than the more generic "date_time".